python/nano/src/bigdl/nano/pytorch/inference/optimizer.py

```diff
@@ -167,6 +173,17 @@ def optimize(self, model: nn.Module,
 
         model.eval()  # change model to eval mode
 
+        input_sample = tuple(next(iter(training_data))[:-1])
```

Comment:
This basically assume users have only one output and one/multiple input for the model's forward, maybe we can inspect the model and set a more accurate param num

Reply:
good catch, will consider this.